### PR TITLE
Biggus, pyke, and ecmwf_grib (iris dependencies)

### PR DESCRIPTION
**Pull request by ocefpaf**
~~Skipping appveyor for now b/c there will be more commits.~~

Ping @pelson and @rhattersley I added you both as maintainers on all recipes in this PR.

### CODE REVIEW

File: recipes/ecmwf_grib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set version = "1.14.7" %}
```
Comment (ocefpaf) [inline recipes/ecmwf_grib/meta.yaml]: Not sure if this version is OK.

**Comment by ocefpaf** [comment]
`Travis-CI`

```
conda.resolve.Unsatisfiable: The following specifications were found to be in conflict:
  - cartopy -> numpy 1.10*|1.9*
  - cartopy -> python 3.4*|3.5*
  - cartopy -> scipy
  - cf_units -> netcdf4
  - cf_units -> python 3.4*|3.5*
  - netcdf4 -> numpy 1.10*|1.5*|1.6*|1.7*|1.8*|1.9*
  - netcdf4 -> python 2.6*|3.3*|3.4*|3.5*
  - python 2.7*
  - scipy -> numpy 1.10*|1.5*|1.6*|1.7*|1.8*|1.9*
  - scipy -> python 2.6*|3.3*|3.4*|3.5*
```

`CircleCI`

```
conda.resolve.Unsatisfiable: The following specifications were found to be in conflict:
  - cartopy -> numpy 1.10*|1.9*
  - cartopy -> python 3.4*|3.5*
  - cartopy -> scipy
  - cf_units -> netcdf4
  - cf_units -> python 3.4*|3.5*
  - netcdf4 -> numpy 1.10*|1.5*|1.6*|1.7*|1.8*|1.9*
  - netcdf4 -> python 2.6*|3.3*|3.4*|3.5*
  - python 2.7*
  - scipy -> numpy 1.10*|1.5*|1.6*|1.7*|1.8*|1.9*
  - scipy -> python 2.6*|3.3*|3.4*|3.5*
```

**Comment by jakirkham** [comment]
Maybe caused by NumPy 1.11 move? Was wondering if we should wait for Continuum to release their NumPy 1.11 before we should switch over.

**Comment by ocefpaf** [comment]
> Maybe caused by NumPy 1.11 move? Was wondering if we should wait for Continuum to release their NumPy 1.11 before we should switch over.

Not sure about that. By forcing `numpy >=1.9` here we get failures that will force us to re-visit the feedstocks and be aware of what is new out there.

BTW: I don't think the issue here was `np111`.

**Comment by ocefpaf** [comment]
@pelson since these are recipes from conda-recipes-scitools do you mind taking a look before we merge?

**Comment by pelson** [comment]
These look very sensible. Thanks for all the hard work on these @ocefpaf! :+1:

**Pull request merged**